apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/onboarding/joining/JoiningNodeInitializer.scala

```diff
@@ -241,12 +241,6 @@ class JoiningNodeInitializer(
               )
             _ <- svStore.domains.waitForDomainConnection(config.domains.global.alias)
             _ <- dsoStore.domains.waitForDomainConnection(config.domains.global.alias)
-            _ <- checkIsOnboardedAndStartSvNamespaceMembershipTrigger(
```

Comment:
Without this the triggers runs twice after a restart (with or without synchronizer initialization) of a joining node. This was introduced ages ago by https://github.com/DACH-NY/canton-network-node/pull/11097.

I'll create a separate tech debt to make sure we never silently start two triggers of the same type for the same automation service.

Reply:
https://github.com/hyperledger-labs/splice/issues/1497